core/trino-main/src/test/java/io/trino/sql/planner/iterative/rule/TestPushProjectionIntoTableScan.java

```diff
@@ -341,4 +335,23 @@ private static ColumnHandle column(String name, Type type)
     {
         return new TpchColumnHandle(name, type);
     }
+
+    private static Map<String, ExpressionMatcher> createAssignments(ImmutableMap<Symbol, String> newNames, ImmutableMap<Symbol, Expression> constants)
```

Comment:
nit: while we love immutable collections and use them everywhere, in the interest of code readability we don't declare fields, variables or parameters as "Immutable...". Just `Map` will be enough

Reply:
This function has been removed now as per the earlier comment to make the code inline.